chart/README.md

```diff
@@ -101,6 +101,7 @@ You can set up the credentials, nodeSelector, volume sizes (default volumes crea
 | `promscale.image`                        | Docker image to use for the Connector                 | `timescale/promscale:0.1.0-alpha.2` |
 | `promscale.connection.dbName`            | Database to store the metrics in                      | `postgres`  |
 | `promscale.connection.user`              | User used for connection to db | `postgres` |
+| `promscale.connection.dbURI.secretTemplate` | The template for generating the name of a secret object which will hold the db URI | `` |
```

Comment:
This will be used by Promscale on startup to connect with db-uri.  Here we can provide promscale the db specific details such as host, port, user, password, or the db-uri. It's a matter of preference. But to make db-uri work as intended & considering recent work over support for db-uri in promscale & it's helm chart. I am using db-uri with promscale to make sure it's working as expected.